server/ansible/Inventory/pbench-server.hosts.example

```diff
@@ -1,9 +1,19 @@
+# Change the [servers] section to the set of hosts where you want to
+# install pbench-server. The various hosts can be running any
+# combinations of RHEL7, RHEL8, or (supported) Fedora versions
+# (currently 28 and 29).  They will also have different configuration
+# files which can be specifified using the value of the cenv variable
+# (which is used to set the value of the
```

Comment:
Why don't they just set `pbench_configuration_environment`?

Reply:
The cenv variable was meant for manual typing on the command line: `pbench_congifuration_environment ` just dosn not roll off my fingers easily :-)